### DISCUSSION TITLE
Complexifying Plausible?

### DISCUSSION THREAD

**Discussion by ErisFY** [plausible/analytics#955] [Anything Else]
I'm guessing this has already been discussed in the past, but are there any plans to complexify Plausible and add more features to it? I would really love to see something like heatmaps, behaviours for a given session, etc be added to Plausible.

**Comment by kerns** [comment]
I can't speak for anyone other than myself (obviously), but I personally appreciate the speed and simplicity of Plausible as it is. If I could add one feature it might be A/B testing, though maybe there is a way to do that already :). 

But if you really need more features or complexity maybe take a look at Matomo?

**Comment by ukutaht** [comment]
I wouldn't hold my breath :) From talking to users, the simplicity of our UI is what people seem to love about the product. We are hesitant to add too many features to the point of looking like GA/Matomo.

There are other tools that focus on heatmaps, session recordings, etc. We're not really trying to occupy the same space.
